db_sql.go

```diff
@@ -42,8 +42,7 @@ const (
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 type sqlUserStoreDB struct {
-	enableAuthenticator bool
-	db                  *sql.DB
+	db *sql.DB
```

Comment:
seriously - enable gofmt so that it runs every time you hit "save". It REALLY is the right way to setup your dev environment. I have tried the other options. That really is best.